src/ffi/pystate.rs

```diff
@@ -21,43 +25,45 @@ extern "C" {
     #[cfg_attr(docsrs, doc(cfg(not(PyPy))))]
     pub fn PyInterpreterState_Delete(arg1: *mut PyInterpreterState);
 
-    #[cfg(all(Py_3_9, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_9), not(PyPy)))]
```

Comment:
This isn't exactly right: on Python 3.8 and below this API would have been called `_PyInterpreterState_Get` (with a leading underscore).

I didn't think it was worth the added complexity of covering this, hence why only enabled on 3.9 and up (where it was immediately added to the stable API).

Reply:
Change as suggested. Thanks!